dotCMS/src/main/java/com/dotmarketing/business/LegacyVersionableFactoryImpl.java

```diff
@@ -0,0 +1,432 @@
+package com.dotmarketing.business;
+
+import com.dotcms.repackage.com.google.common.annotations.VisibleForTesting;
+import com.dotmarketing.beans.Identifier;
+import com.dotmarketing.beans.Inode;
+import com.dotmarketing.beans.VersionInfo;
+import com.dotmarketing.db.DbConnectionFactory;
+import com.dotmarketing.db.HibernateUtil;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.portlets.containers.business.ContainerAPI;
+import com.dotmarketing.portlets.containers.model.Container;
+import com.dotmarketing.portlets.contentlet.model.ContentletVersionInfo;
+import com.dotmarketing.portlets.templates.business.TemplateAPI;
+import com.dotmarketing.portlets.templates.model.Template;
+import com.dotmarketing.util.InodeUtils;
+import com.dotmarketing.util.Logger;
+import com.dotmarketing.util.UtilMethods;
+import com.liferay.portal.model.User;
+import org.apache.commons.beanutils.BeanUtils;
+
+import java.util.*;
+
+import static com.dotcms.util.CollectionsUtils.set;
+
+/**
+ * @deprecated on 09-23-2020. Replaced by {@link VersionableFactoryImpl}
+ *
+ * Implementation class for the {@link VersionableFactory} class.
+ *
+ * @author Will Ezell
+ * @version 1.0
+ * @since Mar 22, 2012
```

Comment:
Codacy found an issue: [It is a good practice to call super() in a constructor](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6214071)